### ISSUE TITLE
Issue in width of body

### ISSUE THREAD

**Issue by jaydeepkhatri** [eggnaut/eggnaut.github.io#3]
Hi Eggnaut.

I found that using the following CSS code will be a good thing to implement & it will not casue the horizontal scroll issue.

```CSS
div#icons {
    margin-top: 80px;
    margin-bottom: 20px;
    text-align: center;
}
```

Thanks :)

**Comment by eggnaut** [comment]
Hello!

I never knew that was an issue 😦 I’m new to css and html 😅

Oh wow thank you! I’ll add it into the css file in like 20 minutes, thank you so much!

**Comment by eggnaut** [comment]
Fixed in commit #b02f7f0
